pkg/pattern/tee_service.go

```diff
@@ -310,6 +316,29 @@ func (ts *TeeService) sendBatch(ctx context.Context, clientRequest clientRequest
 					// Success here means the stream will be processed for both metrics and patterns
 					ts.ingesterAppends.WithLabelValues(clientRequest.ingesterAddr, "success").Inc()
 					ts.ingesterMetricAppends.WithLabelValues("success").Inc()
+
+					labels := make([]string, 0, len(req.Streams))
+					for _, stream := range req.Streams {
+						labels = append(labels, stream.Labels)
+					}
+
+					sp.LogKV(
+						"event", "forwarded push request to pattern ingester",
+						"labels", strings.Join(labels, ", "),
```

Comment:
Same here, it shouldn't be big but it might be worth checking the length of this just in case?